package.json

```diff
@@ -14,7 +14,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "pretest": "jshint .",
+    "posttest": "eslint .",
```

Comment:
When doing red-green-refactor cycle, we don't want to wait for linter every time we run the test suite. Another way of looking at this: first get the implementation right, only then worry about the (coding) style.